### PR TITLE
bpo-38380: Update macOS & Windows builds to SQLite v3.31.1

### PR DESCRIPTION
**Pull request by erlend-aasland**
https://bugs.python.org/issue38380

https://bugs.python.org/issue38380

Automerge-Triggered-By: @zooba

### CODE REVIEW

**Comment by erlend-aasland** [comment]
FYI: The Windows build fails because I forgot to `git tag` the upgrade with `sqlite-3.30.1.0` in https://github.com/python/cpython-source-deps/pull/17. I've notified the maintainers, hoping that one of them can tag it for me, since I lack the permissions to do so (and creating a PR for just one tag is not possible, AFAIK.)

**Comment by erlend-aasland** [comment]
FYI, updated to most recent SQLite3 version. See bug tracker for discussion.

File: Misc/NEWS.d/next/Build/2020-02-28-23-51-27.bpo-38380.TpOBCj.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Update Windows builds to use SQLite 3.31.1
```
Comment (zooba) [inline Misc/NEWS.d/next/Build/2020-02-28-23-51-27.bpo-38380.TpOBCj.rst]: Move this to the `Windows` section (see [previous entries](https://docs.python.org/3/whatsnew/changelog.html#id73))

File: Misc/NEWS.d/next/Build/2020-02-28-23-51-47.bpo-38380.u-ySyA.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Update macOS builds to use SQLite 3.31.1
```
Comment (zooba) [inline Misc/NEWS.d/next/Build/2020-02-28-23-51-47.bpo-38380.u-ySyA.rst]: Move this to the `macOS` section (see [previous entries](https://docs.python.org/3/whatsnew/changelog.html#id73))

File: Misc/NEWS.d/next/Build/2020-02-28-23-51-27.bpo-38380.TpOBCj.rst (hunk 1 above)
Comment (zooba) [reply Misc/NEWS.d/next/Build/2020-02-28-23-51-27.bpo-38380.TpOBCj.rst]: Just a straight `git mv` is fine, there's nothing encoded in the filename/contents that'll matter.

Comment (erlend-aasland) [reply Misc/NEWS.d/next/Build/2020-02-28-23-51-27.bpo-38380.TpOBCj.rst]: Fixed in commit 6a89f6489bc9bfd3c8c31d677f4a8c3fc59f5b0e

File: Misc/NEWS.d/next/Build/2020-02-28-23-51-47.bpo-38380.u-ySyA.rst (hunk 2 above)
Comment (erlend-aasland) [reply Misc/NEWS.d/next/Build/2020-02-28-23-51-47.bpo-38380.u-ySyA.rst]: Fixed in commit 6a89f6489bc9bfd3c8c31d677f4a8c3fc59f5b0e

**Review by zooba — APPROVED** [review]

**Comment by zooba** [comment]
Closing/reopening to retrigger Azure Pipelines

**Pull request merged**